cypress/e2e/user/search-users.feature

```diff
@@ -1,11 +1,10 @@
 Feature: The user list can be searched
 
-    # SKIPPED because of search problems (7 Jan 2025)
-    # Scenario: The user list gets filtered by name
-    #     Given some users exist
-    #     And the user-manager navigated to the user list view
-    #     When the user-manager searches the list by entering a name
-    #     Then only the users whose username or display name contains the search term should be displayed
+    Scenario: The user list gets filtered by name
```

Comment:
unrelated, but turning these back on because the issue with them is now resolved on the backend